core/iwasm/aot/arch/aot_reloc_aarch64.c

```diff
@@ -53,7 +53,12 @@ get_target_symbol_map(uint32 *sym_num)
     return target_sym_map;
 }
 
+#if defined(__APPLE__) && defined(__arm64__)
+#define BUILD_TARGET_AARCH64_DEFAULT "arm64"
```

Comment:
There was similar change in https://github.com/bytecodealliance/wasm-micro-runtime/pull/2570, but the author hasn't addressed the comment yet, could we also apply its change?
Not sure whether `#if (defined(__APPLE__) || defined(__MACH__)) && defined(__arm64__)` is OK?